### ISSUE TITLE
invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by vinayagg** [argoproj/argo#4544]
## Summary 

What happened/what you expected to happen?
Got following stack trace
```
Name:                create-account-km9ns
Namespace:           argo
ServiceAccount:      argo-service-account
Status:              Pending
Created:             Sun Nov 15 17:11:00 -0800 (1 second from now)
Parameters:
  accountId:         test12
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x318 pc=0x1ad3a58]
goroutine 1 [running]:
k8s.io/apimachinery/pkg/apis/meta/v1.(*Time).IsZero(...)
        /Users/acollins8/go/pkg/mod/k8s.io/apimachinery@v0.16.7-beta.0/pkg/apis/meta/v1/time.go:65
github.com/argoproj/argo/cmd/argo/commands.watchWorkflow(0xc0008dd9e0, 0x14, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/commands/watch.go:96 +0x458
github.com/argoproj/argo/cmd/argo/commands.waitOrWatch(0xc0009408c0, 0x1, 0x1, 0x0, 0x0, 0x10100, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/commands/submit.go:245 +0xdb
github.com/argoproj/argo/cmd/argo/commands.submitWorkflows(0xc000161c00, 0x1, 0x1, 0xc0005bfb00, 0xc0001019a0)
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/commands/submit.go:219 +0x53a
github.com/argoproj/argo/cmd/argo/commands.submitWorkflowsFromFile(0xc00043b180, 0x1, 0x8, 0xc0005bfb00, 0xc0001019a0)
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/commands/submit.go:104 +0x24a
github.com/argoproj/argo/cmd/argo/commands.NewSubmitCommand.func1(0xc0005e7b80, 0xc00043b180, 0x1, 0x8)
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/commands/submit.go:65 +0x176
github.com/spf13/cobra.(*Command).execute(0xc0005e7b80, 0xc00043af80, 0x8, 0x8, 0xc0005e7b80, 0xc00043af80)
        /Users/acollins8/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20181021141114-fe5e611709b0/command.go:766 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0005e6000, 0xc000070750, 0xc000879f50, 0x40576f)
        /Users/acollins8/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20181021141114-fe5e611709b0/command.go:852 +0x2ea
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/acollins8/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20181021141114-fe5e611709b0/command.go:800
main.main()
        /Users/acollins8/go/src/github.com/argoproj/argo/cmd/argo/main.go:17 +0x2b
```

## Diagnostics

What Kubernetes provider are you using? 
GKE

What version of Argo Workflows are you running? 

```yaml
Paste a workflow that reproduces the bug, including status:
kubectl get wf -o yaml ${workflow} 
```
This is NOT workflow dependent. I think this happens if I launch a workflow soon after installing argo (within 1-5 seconds)

```
Paste the logs from the workflow controller:
kubectl logs -n argo $(kubectl get pods -l app=workflow-controller -n argo -o name) | grep ${workflow}
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by sarabala1979** [comment]
@vinayagg Can you provide the argo version which you are using?

**Comment by sarabala1979** [comment]
https://github.com/argoproj/argo/issues/3422

**Comment by vinayagg** [comment]
$ argo version
argo: v2.9.1
  BuildDate: 2020-07-03T15:06:40Z
  GitCommit: 6b967d08c0a142aaa278538f2407c28de467262e
  GitTreeState: clean
  GitTag: v2.9.1
  GoVersion: go1.13.4
  Compiler: gc
  Platform: linux/amd64
=============================
image: argoproj/workflow-controller:v2.11.7

**Comment by sarabala1979** [comment]
This issue is fixed in 2.9.4 +. I would recommend to upgrade the Argo cli to latest version 2.11.7

**Comment by sarabala1979** [comment]
@vinayagg Please verify it on the latest version and close this issue
